go-controller/pkg/ovn/ovn.go

```diff
@@ -703,6 +709,7 @@ func (oc *Controller) WatchCRD() {
 			crd := obj.(*apiextension.CustomResourceDefinition)
 			klog.Infof("Deleting CRD %s from cluster", crd.Name)
 			if crd.Name == egressfirewallCRD {
+				close(oc.egressFirewallDNS.stopChan)
 				oc.watchFactory.RemoveEgressFirewallHandler(oc.egressFirewallHandler)
 				oc.egressFirewallHandler = nil
 				oc.watchFactory.ShutdownEgressFirewallWatchFactory()
```

Comment:
I am only shutting down the DNS process, since there is only one and it causes a panic to close a channel more then once, let me see if there is a way to check for channel close